implementation/src/main/java/io/smallrye/mutiny/groups/UniOnEvent.java

```diff
@@ -62,7 +62,9 @@ public UniOnEvent(Uni<T> upstream) {
      *        necessary {@code null} and the third is necessary {@code false} as it indicates a termination
      *        due to a failure.
      * @return the new {@link Uni}
+     * @deprecated Use {@code uni.onTermination().invoke(...)} instead
      */
+    @Deprecated
     public Uni<T> termination(Functions.TriConsumer<T, Throwable, Boolean> consumer) {
```

Comment:
Not related to this PR, but I believe the whole "on" group is going to be deprecated at some point.

Reply:
Yes, `onCancellation()`, etc etc